### DISCUSSION TITLE
New org owners: Daniel Burgener and Patrick LaFontaine

### DISCUSSION THREAD

**Discussion by nikomatsakis** [lalrpop/lalrpop#970] [Announcements]
I'm happy to announce that I've added @dburgener and @Pat-Lafon as new owners of the lalrpop github organization. I've long since 'retired' from actively managing LALRPOP and the two of them have been doing a great job carrying the torch. So please join me in giving them a well-earned congratulations! 👏 👏 👏

**Comment by yannham** [comment]
Well-deserved :slightly_smiling_face:
